pkg/registry/core/replicationcontroller/registry.go

```diff
@@ -74,15 +74,15 @@ func (s *storage) GetController(ctx genericapirequest.Context, controllerID stri
 }
 
 func (s *storage) CreateController(ctx genericapirequest.Context, controller *api.ReplicationController) (*api.ReplicationController, error) {
```

Comment:
same comment about scale here.  Needs a big warning and a followup issue.

Reply:
Fixed